src/main/java/io/sentry/samples/example/CustomBeforeSendCallback.java

```diff
@@ -0,0 +1,28 @@
+package io.sentry.samples.example;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+import io.sentry.Sentry;
+import io.sentry.SentryOptions;
+import io.sentry.SentryEvent;
+import java.io.PrintStream;
+import java.util.Arrays;
+
+@Component
+public class CustomBeforeSendCallback implements SentryOptions.BeforeSendCallback {
+  
+    @Override
+    public SentryEvent execute(SentryEvent event, Object hint) {
+        // Data Scrubbing
```

Comment:
Don't leave these commented in or could cause huge confusion mid-demo if you're wondering why things are dissappearing or not getting grouped the way you'd expect them to.